src/cow-react/modules/swap/pure/ReceiveAmountInfo/index.tsx

```diff
@@ -29,50 +29,62 @@ export function ReceiveAmountInfoTooltip(props: ReceiveAmountInfoTooltipProps) {
   )
 
   return (
-    <styledEl.Box>
-      <div>
+    <Box>
+      <Column>
         <span>
           <Trans>Before fee</Trans>
         </span>
-        <span>
-          {amountBeforeFees} {currency.symbol}
-        </span>
-      </div>
-      <div>
-        {discount ? <styledEl.GreenText>{FeePercent}</styledEl.GreenText> : FeePercent}
+        <TextAmount>
+          {/* // Split amountBeforeFees string into integers and decimals to style them differently (e.g. 1.2345 -> 1.23 45)  */}
+          <span>{amountBeforeFees.split('.')[0]}</span>
+          <span>.</span>
+          <span>{amountBeforeFees.split('.')[1]}</span>
+          <span>{currency.symbol}</span>
```

Comment:
Not available yet as it [has not been merged](https://github.com/cowprotocol/cowswap/pull/1854), but there will be a component for this `<TokenSymbol token={currency} />`